DyAdd/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<resources>
+    <string name="app_name">Calculator</string>
```

Comment:
You should change this to DyAdd as well so the app name on the device is updated!